### ISSUE TITLE
Feature Request: Month Picker + Year Picker

### ISSUE THREAD

**Issue by mpf82** [vitmalina/w2ui#1957]
We already have date, time and datetime picker.

A month picker and a year picker would be nice additions.

We're currently using JQuery UI's Date Picker as a month picker, but it just does not feel that well integrated with the rest of the w2ui components:

![month_picker](https://user-images.githubusercontent.com/10152404/108691952-57e2e400-74fc-11eb-80d8-ee91775c800a.png)

**Comment by mpf82** [comment]
Yeah, I knew that, but the date picker will not "stop" after selecting the month.

I am suggesting new field types:

```
.w2field('month', ...
.w2field('year', ...
```

analogue to the already existing ``date, time, and datetime``.

For ``month`` it's probably be sufficient to use the already existing ``date`` picker, bring up the month view as default (as seen in your screenshot) and just make it stop after selecting the month.

**Comment by vitmalina** [comment]
I did a change when dbl click on a month will select it with current year (same with year). As for `m/yy` find of formats. Possibly it is a good idea, but still struggling how is it better then 2 drop downs ))

**Comment by mpf82** [comment]
One advantage would be to use start+end functionality, just like 2 w2fields of type ``date``.

And one field makes validation and getting/setting of values easier than 2 drop downs.
